### ISSUE TITLE
Request: Build instructions for windows

### ISSUE THREAD

**Issue by joejenniges** [3vi1/IMP#24]
It would be appreciated to have build instructions for windows. I have Qt Creator, but I normally don't develop in C++. 

Thanks!

**Comment by 3vi1** [comment]
Good idea.  I'll try to throw my build notes into a doc, but if you have Qt Creator installed, there's not too much beyond that unless you want to redistribute the exe.
